packages/seacas/libraries/ioss/src/text_mesh/Iotm_DatabaseIO.C

```diff
@@ -643,6 +643,21 @@ namespace Iotm {
     }
   }
 
+  void DatabaseIO::update_block_omissions_from_assemblies()
+  {
+    if (!assemblyOmissions.empty()) {
+      assert(blockInclusions.empty());
```

Comment:
Is this the only place that we verify that if assemblyOmissions is not empty then blockInclusions must be empty?  If so, then this will not be checked in a non-debug build and will also not provide any error message in debug that a user could figure out what is happening...